recipe/patches/script-fix-python-path.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/msvc_project/VSScript/VSScript.vcxproj b/msvc_project/VSScript/VSScript.vcxproj
+index 7432af58..d004d088 100644
+--- a/msvc_project/VSScript/VSScript.vcxproj
++++ b/msvc_project/VSScript/VSScript.vcxproj
+@@ -79,8 +79,8 @@
+     <LibraryPath>C:\Program Files %28x86%29\Python310-32\libs;$(LibraryPath)</LibraryPath>
+   </PropertyGroup>
+   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+-    <IncludePath>../../include;C:\Program Files\Python310\include;$(IncludePath)</IncludePath>
+-    <LibraryPath>C:\Program Files\Python310\libs;$(LibraryPath)</LibraryPath>
++    <IncludePath>../../include;$(PREFIX)\include;$(IncludePath)</IncludePath>
```

Comment:
```suggestion
+    <IncludePath>../../include;$(LIBRARY_INC);$(IncludePath)</IncludePath>
```
I'm not sure if this works. Please review.

Reply:
This is to include `Python.h` which is in `%PREFIX%\include` instead of `%LIBRARY_INC%` (%PREFIX%\Library\include).